plugins/performance-lab/includes/server-timing/class-perflab-server-timing.php

```diff
@@ -288,19 +288,43 @@ function ( string $output, ?int $phase ): string {
 	 */
 	private function format_metric_header_value( Perflab_Server_Timing_Metric $metric ): ?string {
 		$value = $metric->get_value();
-
-		// If no value is set, make sure it's just passed through.
-		if ( null === $value ) {
-			return null;
-		}
+		$start = $metric->get_start_time();
+		$desc  = $metric->get_description();
 
 		if ( is_float( $value ) ) {
 			$value = round( $value, 2 );
 		}
 
+		if ( is_float( $start ) ) {
+			$start = round( $start, 2 ) * 1000.00;
+		}
+
+		// If no value is set, make sure it's just passed through.
+		if ( null === $start && null === $value ) {
+			return null;
```

Comment:
should this return `''` or the original passed value? Looking at the use of the return value, [this](https://github.com/WordPress/performance/blob/ec4559e69448a500b836379ddf56b269875dafa9/plugins/performance-lab/includes/server-timing/hooks.php#L35) check explicitly looks for an empty string.

Reply:
That's for the whole header though, and this is just for 1 metric, which is a small part of the whole header.

Also note that this is just moved code. It was already returning null before.